controls/audit_log_config.rb

```diff
@@ -5,27 +5,39 @@
 control 'windows-audit-100' do
   impact 0.1
   title 'Configure System Event Log (Application)'
-  desc 'Only appies for Windows 2008 and newer'
+  desc 'Only applies for Windows 2008 and newer'
   describe registry_key('HKLM\Software\Policies\Microsoft\Windows\EventLog\Application') do
     it { should exist }
     its('MaxSize') { should_not eq nil }
   end
+  ## Win7
+  describe registry_key('HKLM\System\CurrentControlSet\Services\EventLog\Application') do
```

Comment:
For whatever reason, you test a different entries that the one I found in the litterature and matched wevtutil changes.
quoted here for example https://msdn.microsoft.com/en-us/library/windows/desktop/aa363648(v=vs.85).aspx
When I ran my ansible role and did the test, it was matched by the GUI.
Not sure which one is the reference now or if depends on windows version.